static/js/components/CommentListMobile.jsx

```diff
@@ -246,144 +314,175 @@ const CommentListMobile = ({ isCandidate }) => {
       <div className={styles.commentsList}>
         {comments
           .slice(0, 3)
-          .map(({ id, author, created_at, text, attachment_name, groups }) => (
-            <span
-              key={id}
-              className={commentStyle}
-              onMouseOver={() =>
-                handleMouseHover(id, userProfile, author.username)
-              }
-              onMouseOut={() => handleMouseLeave()}
-              onFocus={() => handleMouseHover(id, userProfile, author.username)}
-              onBlur={() => handleMouseLeave()}
-            >
-              {compactComments ? (
-                <div className={styles.compactContainer}>
-                  <div className={styles.commentUserAvatar}>
-                    <UserAvatar
-                      size={24}
-                      firstName={author.first_name}
-                      lastName={author.last_name}
-                      username={author.username}
-                      gravatarUrl={author.gravatar_url}
-                    />
-                  </div>
-                  <div className={styles.compactWrap} name={`commentDiv${id}`}>
-                    <ReactMarkdown
-                      source={text}
-                      escapeHtml={false}
-                      className={styles.commentMessage}
-                      renderers={{ text: emojiSupport }}
-                    />
-                    <div className={styles.compactButtons}>
-                      <Tooltip
-                        title={dayjs().to(dayjs.utc(`${created_at}Z`))}
-                        placement="left"
-                      >
-                        <InfoOutlinedIcon fontSize="small" />
-                      </Tooltip>
-                      <div className={styles.spacer}>
-                        <Button
-                          style={
-                            hoverID === id
-                              ? {
-                                  display: "block",
-                                  minWidth: "0",
-                                  lineHeight: "0",
-                                  padding: "0",
-                                }
-                              : { display: "none" }
-                          }
-                          size="small"
-                          color="primary"
-                          name={`deleteCommentButton${id}`}
-                          onClick={() => {
-                            dispatch(sourceActions.deleteComment(id));
-                          }}
-                          className="commentDelete"
+          .map(
```

Comment:
What changed exactly in this portion of the diff?

Reply:
The major addition is the spectrum_id and api_path (the latter will be removed I think). Then I use them when rendering the text, the attachment, and the delete button.